module/x/gravity/abci.go

```diff
@@ -39,6 +40,22 @@ func createValsets(ctx sdk.Context, k keeper.Keeper) {
 	}
 }
 
+func createBatches(ctx sdk.Context, k keeper.Keeper) {
+	// Auto BatchRequest Creation.
+	// This function automatically requests batches of all token types every 5 blocks
+	// thanks to the more profitable batches rule (see batch creation spec) new batches
+	// will not be created unless the sum of their fees is greater than the previous batch
+	// that has not yet been executed. Eventually batches will either time out, or a
+	// batch profitable enough to submit will be generated.
+	if ctx.BlockHeight()%5 == 0 {
+		tokens := k.GetEthereumOriginatedErc20Tokens(ctx)
```

Comment:
Why only ethereum originated?

Reply:
Cosmos originated assets that have been bridged are a separate list, figured I would test this first but then ran out of time.